### ISSUE TITLE
The basic process gets "killed" after/during "Calculating clusters for the following combination of bands"

### ISSUE THREAD

**Issue by mazingaro** [cordmaur/WaterDetect#9]
Input (ran from the jupyter notebook, so i leave the original `!`):
`
!waterdetect -i {input_folder} -o {output_folder}`

Output:
```
Loading configuration file WaterDetect.ini
File WaterDetect.ini verified.
Folder / verified.
Folder /waterdetect_test verified.
Opening image in folder /waterdetect_test
Retrieving bands for product S2_THEIA
Opening image in folder /SENTINEL2A_20180824-100141-227_L2A_T33TVL_C_V1-0
Retrieving bands for product S2_THEIA
10 were found:
Loading band: Blue
Loading band: Green
Loading band: Red
Loading band: Mir
Loading band: Mir2
Loading band: Nir
Loading band: Nir2
Loading band: RedEdg1
Loading band: RedEdg2
Loading band: RedEdg3
0...10...20...30...40...50...60...70...80...90...100 - done.
Input file size is 10980, 10980
0...10...20...30...40...50...60...70...80...90...100 - done.
Saving image: /SENTINEL2A_20180824-100141-227_L2A_T33TVL_C_V1-0_invalid_mask.tif
Calculating clusters for the following combination of bands:
['mndwi', 'ndwi', 'Mir2']
Killed
```
The process stops. This are the files it creates (and apparently copies?)
![image](https://user-images.githubusercontent.com/15820322/113932618-1c724f80-97f4-11eb-8fab-95c3a9f27d41.png)

**Comment by cordmaur** [comment]
@mazingaro,
Good to Know you solved the GDAL issue. I will check for sklearn version on the next release to import the correct name. 

Concerning the process stop during clustering it is probably an out of memory issue. For full resolution sentinel 2 it requires 32Gb, ou you can use a shape file as input and it will crop to that region. 

Can monitor your RAM usage?

**Comment by mazingaro** [comment]
@cordmaur 
Ok, I will let you know. Yes, if you say so then it is probably this: I was testing the script on a laptop with 8 GB of RAM.

Thank you!

**Comment by cordmaur** [comment]
Hi @mazingaro,

I've published a new release (1.5.6) and I (hopefully) corrected the GDAL issue. 
There is a changelog with the main updates (new features also).
If you have any further problem, please let us know.
